src/Chess.cpp

```diff
@@ -0,0 +1,257 @@
+	#include "Chess.h"
+
+// clear the screen "cls"
+void Chess::clear() const 
```

Comment:
this class seems to have multiple responsibilities, such as managing the game state, handling user input, and displaying the board. Consider separating these responsibilities into different functions or classes to follow the single responsibility principle.